libhif/hy-goal.c

```diff
@@ -310,6 +359,12 @@ list_results(HyGoal goal, Id type_filter1, Id type_filter2, GError **error)
                                  HIF_ERROR_INTERNAL_ERROR,
                                  "no solv in the goal");
             return NULL;
+        } else if (goal->removal_of_protected->len) {
+            g_set_error_literal (error,
+                                 HIF_ERROR,
+                                 HIF_ERROR_REMOVAL_OF_PROTECTED_PKG,
+                                 "no solution, protected package trying to be removed");
```

Comment:
There's definitely something wrong with that phrase. Are you trying to say something like: "Cannot remove protected package"?